cockatrice/src/tab_game.cpp

```diff
@@ -309,6 +318,7 @@ TabGame::TabGame(TabSupervisor *_tabSupervisor, GameReplay *_replay)
     sayLabel(0),
     sayEdit(0)
 {
+    MainWindow = new QMainWindow;
```

Comment:
Yes; since MainWindow is a children object of TabGame, when TabGame will be deleted it will take care of deleting MainWindow, too; source: http://doc.qt.io/qt-4.8/objecttrees.html